workflows-scripts/inventory/view-bibs/jmeter-supported-data/keywords.csv

```diff
@@ -0,0 +1,1356 @@
+"A. A",2
```

Comment:
How did this file generate the loop count? Is it random?

Reply:
Will it loop over this file once it reaches the end of the file if the duration of the test runs longer than this list can supply the words?

Reply:
Hello @mtraneis, It is not random it SQL script select substring(title, 1,4), count(1) -2 from holdings_record group by substring(title, 1,4) having count(1) >3 and count(1)<7.